nixos/modules/services/networking/unifi.nix

```diff
@@ -7,6 +7,7 @@ let
     @${cfg.jrePackage}/bin/java java \
         ${optionalString (cfg.initialJavaHeapSize != null) "-Xms${(toString cfg.initialJavaHeapSize)}m"} \
         ${optionalString (cfg.maximumJavaHeapSize != null) "-Xmx${(toString cfg.maximumJavaHeapSize)}m"} \
+        -Dlog4j2.formatMsgNoLookups=true \
```

Comment:
one optional nit though: set this parameter only if the unifi package is known to be vulnerable (eg versionBefore "6.5.54")